core/db/migrate/20250120094216_create_page_builder_models.rb

```diff
@@ -14,7 +14,6 @@ def change
         t.datetime :deleted_at
 
         t.index ['deleted_at'], name: 'index_spree_themes_on_deleted_at'
-        t.index ['store_id', 'default'], name: 'index_spree_themes_on_store_id_and_default', unique: true, where: "((deleted_at IS NULL) AND (\"default\" = true))"
```

Comment:
We don't need the `Spree::Theme` validation for this - we have a callback for ensuring only one theme is default: https://github.com/spree/spree/blob/main/core/app/models/spree/theme.rb#L180